tests/integration/preprocessor/_regrid/test_regrid.py

```diff
@@ -50,6 +50,27 @@ def test_regrid__linear(self):
         expected = np.array([[[1.5]], [[5.5]], [[9.5]]])
         self.assertArrayEqual(result.data, expected)
 
+    def test_regrid__linear_extrapolate(self):
```

Comment:
I think this function should look more like `test_regrid__linear_extrapolate_with_mask`, i.e. you should check that the extrapolated points are as expected. With the current choice of target grid, no extrapolation is needed.

Reply:
good point!

Reply:
@valeriupredoi Did you work on this already? This still needs to be done this before this PR is ready to be merged.

Reply:
whoops, nops, looking at it right now

Reply:
Don't forget that this regridding scheme should not produce any masked values, but do linear extrapolation instead.

Reply:
doh! cheers, man I was wondering what the actual difference between with mask and no mask was :rofl:

Reply:
it now has a big fat 3x3 array, no masking, I promise :grin: